src/main/java/uk/gov/justice/hmpps/casenotes/services/MergeOffenderService.java

```diff
@@ -20,10 +20,9 @@
     public int checkAndMerge(final Long bookingId) {
         final var rowsUpdated = new AtomicInteger();
         log.debug("Check for merged booking for ID {}", bookingId);
-        externalApiService.getIdentifiersByBookingId(bookingId).stream()
-                .filter(id -> "MERGED".equals(id.getType()))
+        final var booking = externalApiService.getBooking(bookingId);
+        externalApiService.getMergedIdentifiersByBookingId(bookingId)
                 .forEach(id -> {
-                    final var booking = externalApiService.getBooking(bookingId);
```

Comment:
don't need to get the booking for each merged identifier, since it will be the same